sites/fast-component-explorer/app/fast-components/configs/fast-anchor.ts

```diff
@@ -14,6 +20,35 @@ const fastAnchorConfig: ComponentViewConfig = {
                     root: {
                         schemaId: fastAnchorId,
                         data: {
+                            href: "#",
+                            Slot: [
+                                {
+                                    id: "Slot",
+                                },
+                            ],
+                        },
+                    },
+                    Slot: {
```

Comment:
should these all be capitalized?

Reply:
Yes, this is how Jane defined them in the mappings.

https://github.com/microsoft/fast-dna/blob/763054f36433f9b87e620c0c8d03c229dcd8560f/packages/tooling/fast-tooling/src/data-utilities/mapping.ts#L434